### PR TITLE
Rename interaction variables

### PR DESCRIPTION
**Pull request by marcjansen**
There are more renames, but this is hard to look at already.

Please review.

### CODE REVIEW

File: examples/snap.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ Modify.init();
 
 var optionsForm = document.getElementById('options-form');
 
-var Draw = {
+var ExampleDraw = {
```
Comment (tschaub) [inline examples/snap.js]: Curious if the linter caught this after rename, if you searched for 'var Draw' before, or if something else.

**Comment by fredj** [comment]
rebased

**Comment by tschaub** [comment]
Hrm.  That was a failed experiment using GitHub's web-based conflict editor.  I'll fix this up...

**Comment by tschaub** [comment]
@fredj - I'm curious to hear how you did the rebase.  GitHub is letting me modify this branch through the web editor, but I can't push directly (and I failed at conflict resolution with the web editor).

I opened #7607 with a rebased version of @marcjansen's commits.  I'll close this PR in favor of that one.

File: examples/snap.js (hunk 1 above)
Comment (marcjansen) [reply examples/snap.js]: The linter warns, as the variable was already used.

**Comment by fredj** [comment]
@tschaub I've rebased locally on the pushed force to Marc's repository.

**Pull request closed without merging**